WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewModel.swift

```diff
@@ -249,6 +258,8 @@ final class CardReaderSettingsConnectedViewModel: CardReaderSettingsPresentedVie
             newShouldShow = .isUnknown
         } else if connectedReaders.isEmpty {
             newShouldShow = .isFalse
+        } else if connectedReaders.first(where: { $0.readerType == .appleBuiltIn }) != nil {
```

Comment:
Technically this means that if any of the connected readers is the Apple Built-in one we don't show the view right? What happens if we are connected to a Bluetooth reader apart from of that?

Reply:
Correct, however we don't actually support being connected to more than one reader, and [neither do Stripe](https://stripe.dev/stripe-terminal-ios/docs/Classes/SCPTerminal.html#/c:objc(cs)SCPTerminal(py)connectedReader).

So we can't be connected to more than one at a time, and this class doesn't otherwise handle that situation anyway (it relies on `.first` to determine the connected reader.)

Reply:
The proper fix for this inconsistency would be to change `connectedReaders` to a single object, not an array, but that is way out of scope here 😊

Reply:
> The proper fix for this inconsistency would be to change connectedReaders to a single object, not an array, but that is way out of scope here 😊

That's what I thought, can we create an issue to change that?